CHANGELOG.md

```diff
@@ -6,9 +6,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.16.7]
+### Fixed
+- Fix bug that breaks dapps that expect users to switch chains manually([#25046]https://github.com/MetaMask/metamask-extension/pull/25046)
+
+## [11.16.6]
```

Comment:
I could be understanding it wrongly, since we never released 11.16.6, should we move changelog to 11.16.7?

Reply:
No, I think we should retroactively create a 11.16.6 releases page. The release does exist, even if we didn't release it